app/assets/scss/_brief_submission.scss

```diff
@@ -52,6 +52,13 @@ ol.instruction-list-item-top li {
 }
 
 ol.steps {
+  @include bold-27;
+  padding-left: 1em; // 27px
+
+  @include media( $min-width: $site-width + (2 * (27px + $gutter) ) ) {
+    padding-left: 0;
+  }
+
```

Comment:
We had a chat about this a while ago.

History of it is that the styles Dan took out of `application.scss` we added when there was only one use of the instruction list pattern in this app, on the brief overview page (which uses the `steps` class for the list of things you need to do to publish the brief). This meant all instruction lists in the app had a 27pt font.

Dan's change puts those styles in the block tied to the `steps` class so tying it to the overview page. This mean our use of the list, in this page, reverted to those defined in [the original pattern](http://alphagov.github.io/digitalmarketplace-frontend-toolkit/instruction-list.html).

In other words, sensible stuff 👍